### PR TITLE
ENG-5012 created system check script

### PR DESCRIPTION
**Pull request by dweiss88**

### CODE REVIEW

File: lib/python/voltcli/voltdb.d/check.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                output['OS release'] = ["WARN", "Unsupported release: " + osRelease]
+        elif "red hat" in osRelease.lower() or "rhel" in osRelease.lower():
+            if float(release) >= 6.3:
+                output['OS release'] = ["PASS", osRelease]
+            else:
+                output['OS release'] = ["WARN", "Unsupported release: " + osRelease]
+        else:
+            output['OS release'] = ["WARN", "Unsupported release: " + osRelease]
+    else:
+        osName=["WARN", "Supported platforms are MAC or Linux based"]
```
Comment (akhanzode) [inline lib/python/voltcli/voltdb.d/check.py]: We don't support MAC this script should not print MAC as supported platform.

File: lib/python/voltcli/voltdb.d/collect.py [hunk 2]
```diff
@@ -63,6 +63,13 @@ def collect(runner):
     if int(runner.opts.days) == 0:
 	print >> sys.stderr, "ERROR: '0' is invalid entry for option --days"
         sys.exit(-1)
+
+    os.environ["PATH"] += os.pathsep + os.pathsep.join(s for s in sys.path if os.path.join("voltdb", "bin") in s)
+    checkFD = os.open(os.path.join(runner.opts.voltdbroot, "systemcheck"), os.O_WRONLY|os.O_CREAT|os.O_TRUNC)
+    checkOutput = os.fdopen(checkFD, 'w')
+    subprocess.call("voltdb check", stdout=checkOutput, shell=True)
```
Comment (akhanzode) [inline lib/python/voltcli/voltdb.d/collect.py]: Do you need to call a new process? Can it not invoke check.systemcheck?

**Comment by akhanzode** [comment]
Can you add a test in TestCollector to include systemcheck?

File: lib/python/voltcli/voltdb.d/check.py [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+        osName= "Mac OS X"
+        # support for macosx 10.7 11.0.0 and greater
+        osRelease = os.popen("sw_vers -productVersion | tr -d '\n'").read()
+        output['OS'] = ["PASS", osName]
+        output['OS release'] = ["PASS", osRelease]
+    elif isHostLINUX():
+        osName = "Linux"
+        output['OS'] = ["PASS", osName]
+        # support for specific ubuntu centos
+        osRelease = os.popen("lsb_release -d | tr -s '\t' ' ' | cut -d' ' -f2- | tr -d '\n'").read()
```
Comment (pdshaw) [inline lib/python/voltcli/voltdb.d/check.py]: subprocess.Popen replaces os.popen, and is the preferred package.

**Pull request merged**